pw-3/src/1-task.ts

```diff
@@ -0,0 +1,17 @@
+type AddToSum = (number2: number) => number | AddToSum;
+
+export const add = function (number: number): AddToSum {
+	let sum = number;
+
+	function addToSum(number2?: number): AddToSum | number {
+		if (number2 !== undefined) {
+			sum += number2;
+
```

Comment:
better to add something like prettier for consistent styling

Reply:
Replaced 4 spaces in tab by 2 and ensured tabs to be consisting of spaces instead of symbols to display properly on GitHub